src/rgw/motr/gc/gc.cc

```diff
@@ -47,22 +47,42 @@ void *MotrGC::GCWorker::entry() {
         << worker_id << " Working on GC Queue: " << iname << dendl;
       // time based while loop
       do {
-        // fetch the next entry from index "iname"
-        // check if the object is ready for deletion
-        // for the motr object
-
-        // delete that object
-        // rc = dequeue();
-
-        // remove the entry from iname
+        bufferlist bl;
+        std::vector<std::string>keys(motr_gc->max_count + 1);
+        std::vector<bufferlist>vals(motr_gc->max_count + 1);
+        uint32_t rgw_gc_obj_min_wait = cct->_conf->rgw_gc_obj_min_wait;
```

Comment:
Hi, in enqueue() I have added min_wait_time to the 1_ key. so we need to remove it from one of the places.

Reply:
We have two options here
We can remove "1_" from "1_ExpiryTime", convert it from string to time and then compare
Or we have `time` attribute in `motr_gc_obj_info`, we can add `rgw_gc_obj_min_wait` to it.

I have choosed the 2nd one as it looks cleaner, let me know your thoughts.

Reply:
Hi agreed, the second approach definitely looks better.  👍
Just one suggestion we should change the variable name to `deletion_time` instead of `time` so that we know it is not the expiration time.

Reply:
Updated `time` to `deletion_time`